corehq/apps/app_manager/templates/app_manager/v1/partials/case_list.html

```diff
@@ -90,23 +90,34 @@
                     <th></th>
                     <th>
                         {% trans "Sort Property" %}
-                        <span style="font-size: 13px">
+                        <span class="hq-help-template"
+                              data-title="{% trans "Sort Properties" %}"
+                              data-content=
+                                  "{% blocktrans %}
+                                      Properties in this list determine how
+                                      cases are ordered in your case list. This
+                                      is useful if for example you want higher
+                                      priority cases to appear closer to the
+                                      top of the list. The case list will sort
+                                      by the first property, then the second,
+                                      etc.
+                                  {% endblocktrans %}" >
+                        </span>
+                    </th>
+                    <th>{% trans "Direction" %}</th>
+                    <!-- ko if: hqImport('app_manager/js/app_manager.js').checkCommcareVersion("2.35") -->
```

Comment:
At some point it was best practice to put version checks for features in https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/feature_support.py, as a way to document that feature's dependency on that version. Not sure if we abandoned that on purpose, but I still like that convention.